### PR TITLE
smoke test hook

### PR DESCRIPTION
**Pull request by seankane-msft**
Integrates `eng/tools/smoketests` into CI pipelines

### CODE REVIEW

File: eng/pipelines/templates/jobs/new.smoke.tests.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - name: TimeoutInMinutes
+    type: number
+    default: 60
+
+jobs:
+  - job: SmokeTest
+    steps:
+      - ${{ if eq(parameters.Daily, true) }}:
+        - task: PowerShell@2
+          displayName: 'Run Nightly SmokeTests'
```
Comment (benbp) [inline eng/pipelines/templates/jobs/new.smoke.tests.yml:17]:
```suggestion
          displayName: 'Run Daily SmokeTests'
```

File: eng/pipelines/templates/jobs/new.smoke.tests.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            filePath: eng/scripts/Smoke_Tests_Daily.ps1
+            arguments: '.'
+            pwsh: true
+
+      - ${{ if eq(parameters.Daily, false) }}:
+        - task: PowerShell@2
+          displayName: 'Run Pre-Release SmokeTests'
+          inputs:
+            targetType: 'filePath'
+            filePath: $(Build.SourcesDirectory)/eng/scripts/Smoke_Tests_Daily.ps1
```
Comment (benbp) [inline eng/pipelines/templates/jobs/new.smoke.tests.yml:29]:
```suggestion
            filePath: $(Build.SourcesDirectory)/eng/scripts/Smoke_Tests_Release.ps1
```

Comment (benbp) [inline eng/pipelines/templates/jobs/new.smoke.tests.yml:29]: Is there a versioning difference here or just a service directory target difference? Should the script just be called "Smoke_Tests.ps1"?

Comment (seankane-msft) [reply eng/pipelines/templates/jobs/new.smoke.tests.yml:29]: I can make that name change

**Review by ckairen — APPROVED** [review]

**Review by benbp — APPROVED** [review]

**Pull request closed without merging**